external-crates/move/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -787,7 +787,13 @@ fn var(v: Var) -> IR::Var {
 }
 
 fn field(f: Field) -> IR::Field {
-    sp(f.0.loc, IR::Field_(f.0.value))
+    // If it's a positional field, lower it into `pos{field_idx}` so they're a valid identifier
+    let field_ident = if f.0.value.parse::<u8>().is_ok() {
```

Comment:
Is `f.0.value.starts_with('0'..'9')` not sufficient? 

This seems fine... just something about parsing it scares me lol